docker/Dockerfile

```diff
@@ -19,9 +22,13 @@ RUN apt-get update && apt-get upgrade -y && \
 
 RUN mkdir -p /opt/wonderful/
 ADD https://wonderful.asie.pl/bootstrap/wf-bootstrap-x86_64.tar.gz /opt/wonderful/
+ADD https://wonderful.asie.pl/bootstrap/wf-bootstrap-aarch64.tar.gz /opt/wonderful/
+# Note that the downloads are cached; no impact if it's a multi-platform build.
+# Needed because Docker's TARGETARCH doesn't match Linux's `uname -m` format.
```

Comment:
One way to avoid this is to create symlinks on Asie's server:
wf-bootstrap-amd64.tar.gz -> wf-bootstrap-x86_64.tar.gz
and
wf-bootstrap-arm64.tar.gz -> wf-bootstrap-aarch64.tar.gz

They have to be symlinks, not hard links (a symlink changes when the original changes, a hard link doesn't).

Then replace the two `ADD`s with one single:
` ADD https://wonderful.asie.pl/bootstrap/wf-bootstrap-$TARGETARCH.tar.gz /opt/wonderful/ `

It only saves 8MB on the image but it'd be the proper/official way to do it.

Reply:
I'll take the bandwidth hit for now.